### PR TITLE
www-client/chromium: 112.0.5615.49

### PR DESCRIPTION
**Pull request by PF4Public**
Haven't finished building yet, so `keeplibs` might need adjusting.

PS: Hope [it wasn't April Fools' Day joke](https://gitweb.gentoo.org/data/glep.git/commit/glep-0076.rst?id=9a01af44a438cc3ceda548cab6c7834cf51aa222).

### CODE REVIEW

File: www-client/chromium/chromium-112.0.5615.49.ebuild [hunk 1]
```diff
@@ -0,0 +1,1256 @@
...
+	rm "${WORKDIR}"/patches/chromium-111-v8-std-layout1.patch || die
+	rm "${WORKDIR}"/patches/chromium-111-v8-std-layout2.patch || die
+
+	local PATCHES=(
+		"${WORKDIR}/patches"
+		"${FILESDIR}/chromium-98-gtk4-build.patch"
+		"${FILESDIR}/chromium-108-EnumTable-crash.patch"
+		"${FILESDIR}/chromium-109-system-zlib.patch"
+		"${FILESDIR}/chromium-109-system-openh264.patch"
+		"${FILESDIR}/chromium-111-ozone-platform.patch"
```
Comment (stha09) [inline www-client/chromium/chromium-112.0.5615.49.ebuild:344]: The patch is included in M112.

File: www-client/chromium/chromium-112.0.5615.49.ebuild [hunk 2]
```diff
@@ -0,0 +1,1256 @@
...
+	# disable global media controls, crashes with libstdc++
+	sed -i -e \
+		"/\"GlobalMediaControlsCastStartStop\",/{n;s/ENABLED/DISABLED/;}" \
+		"chrome/browser/media/router/media_router_feature.cc" || die
+
+	rm "${WORKDIR}"/patches/chromium-110-dpf-arm64.patch || die
+	rm "${WORKDIR}"/patches/chromium-111-v8-std-layout1.patch || die
+	rm "${WORKDIR}"/patches/chromium-111-v8-std-layout2.patch || die
+
+	local PATCHES=(
```
Comment (stha09) [inline www-client/chromium/chromium-112.0.5615.49.ebuild:338]: Which USE flags/env did you use? It is very unlikely that no patches are required with libstdc++ and/or GCC.

Comment (PF4Public) [reply www-client/chromium/chromium-112.0.5615.49.ebuild:338]: That is absolutely correct! I did just a version bump and like other chromium ebuilds this likely won't be built by gcc.

I use clang and it builds well.

File: www-client/chromium/chromium-112.0.5615.49.ebuild (hunk 1 above)
Comment (PF4Public) [reply www-client/chromium/chromium-112.0.5615.49.ebuild:344]: Sorry, my bad.

File: www-client/chromium/chromium-112.0.5615.49.ebuild (hunk 2 above)
Comment (thesamesam) [reply www-client/chromium/chromium-112.0.5615.49.ebuild:338]: Can you test it with GCC then?

**Pull request closed without merging**